zcash_primitives/src/transaction/builder.rs

```diff
@@ -723,21 +831,30 @@ mod testing {
         transaction::Transaction,
     };
 
-    impl<'a, P: consensus::Parameters, R: RngCore> Builder<'a, P, R, ()> {
-        /// Creates a new `Builder` targeted for inclusion in the block with the given height
-        /// and randomness source, using default values for general transaction fields.
+    impl<'a, P: consensus::Parameters> Builder<'a, P, ()> {
+        /// Creates a new `Builder` targeted for inclusion in the block with the given height,
+        /// using default values for general transaction fields.
         ///
         /// # Default values
         ///
         /// The expiry height will be set to the given height plus the default transaction
         /// expiry delta.
         ///
         /// WARNING: DO NOT USE IN PRODUCTION
-        pub fn test_only_new_with_rng(
+        pub fn test_only_new(
```

Comment:
This function can go away entirely now, because the builder no longer has an `R` parameter. Conversely, `mock_build` now needs to be named and documented _much_ more scarily, as it is now reachable on regular builders.